### ISSUE TITLE
Create a template for new issues

### ISSUE THREAD

**Issue by dalthviz** [spyder-ide/qtpy#396]
Maybe it could be worthy to add some instructions to submit an issue including asking for the basic setup and other info needed to reproduce if there is an error with QtPy like the steps to reproduce, QtPy version, binding(s) version, OS, Python version, etc (maybe we could use some of the questions at https://github.com/spyder-ide/qtpy/issues/395#issuecomment-1368714320).

What do you think @CAM-Gerlach @ccordoba12 ?

**Comment by ccordoba12** [comment]
I think this is a good idea, so I'm +1 for it.

**Comment by CAM-Gerlach** [comment]
Yeah, definitely a good idea—I can help here. We should have separate issue templates for bug reports, feature requests and generic (blank) issues, and maybe add a button to the issue chooser to redirect users who have issues with installing/running Qt or the bindings themselves unrelated to QtPy.
